usersimcrs/simulator/agenda_based/interaction_model.py

```diff
@@ -2,12 +2,14 @@
 
 import os
 import random
-from typing import Dict, List, Tuple
+from typing import Any, Dict, List, Tuple
 
 import yaml
 from dialoguekit.core.intent import Intent
 from dialoguekit.participant import DialogueParticipant
 
+INTENT_DISTRIBUTION = Dict[Intent, Dict[Intent, int]]
```

Comment:
Should be named `IntentDistribution` (assuming it's a type alias and not a const)

Reply:
Done.